series/plzen-pyvo/events/2018-01-10.yaml

```diff
@@ -0,0 +1,29 @@
+city: plzen
+start: 2018-01-10 18:00:00
+name: Plzeňské Pyvo
+number: 1
+topic: Běhová prostředí
+venue: beer-factory
+description: |
+  Zdravíme všechny západočeské příznivce Pythonu!
+
+  Pohled na mapu srazů Python komunity vypadá pro Čechy dost chmurně, ale dle našich informací vůbec neodpovídá realitě! Tak to pojďme napravit, setkat se na Plzeňském Pyvu a ukázat Moravákům, zač je toho loket. :)
+
+  Bude co pít, bude co jíst a hlavně – bude co poslouchat. Na první sraz připravíme dvě přednášky z oblasti běhových prostředí, jednu vyladíme pro začátečníky a druhou spíše pro pokročilé. Poté samozřejmě dáme prostor pro oblíbené Lightning talky a vše zakončíme volnou diskuzí a networkingem. **Start přednášek bude v 18:30.**
+
+  Doražte včas, ať si máte kam sednout. ;) Těšíme se na Vás!
+
+talks:
+  - title: Úvodní slovo
+    speakers:
+      - Filip Vaculík
+      - Tomáš Bedřich
+      - Jiří Bartoň
+  - title: 'Začátečníci: Virtualenv – co to je a jak to funguje?'
+    speakers:
+      - Tomáš Bedřich
+  - title: 'Pokročilí: Google App Engine pro běh Pythoních aplikací'
```

Comment:
Budu se snažit, aby i tohle mohlo být pro začátečníky užitečné. 

Možná by to sneslo malé doplnění názvu:
"...pro běh Pythoních aplikací online." nebo  "...pro běh Pythoních aplikací v Internetu."
Jsem ze staré školy. Píšu Internet s velkým I protože to považuji za vlastní jméno této jediné sítě. Na rozdíl od internetu, jakékoliv mezinárodní sítě.

Reply:
Týjo, malé velké I u slova internet! Já jsem z tak nové školy, že si ze slova internet děláme srandu a píšeme ho zásadně už jen jako "tyhlety internety" 😄

Reply:
Jinak když už jsme u toho, "běh aplikací v Internetu" mi moc nezní, asi jsem zvyklý spíše na "aplikací na Internetu". Ale může to být rovněž nějaké barbarské novum, takže mě klidně ignorujte.

Reply:
Internet je mužského neživotného, tak pro to. Celé by to bylo "...pro běh Pythoních aplikací v prostředí Internetu."

Reply:
Já odbočím: **pythonních** se píše s malým p (přídavné jméno) a dvěma n (není to primárně podle pravidel pro zvířata). Obecně je to tvar, kterému se chceš vyhnout, pokud to jde, ale pokud to nejde, aspoň ho píšeš správně. K tomuto konsenzu jsme došli po dlouhém vyjednávání, není to jen z mé hlavy.

Reply:
Díky za korekci. Zní to rozumnňně :)

Reply:
Kluci, vy jste hnidopiši (snad jsem to napsal správně). 😁 Kdybych tam udělal nějakou hrubku nebo faktickou chybu, tak neřeknu, ale tohle.. 🤷‍♂️

Reply:
Fixnuto v 230d03d (PR #57)

Reply:
Ber to jako průvodní jev toho, že to máš správně. Když tam nemáš chyby, tak nám nezbývá, než řešit hnidopichy. Kdybychom k tomu nic neměli, tak bys mohl mít pocit, že jsme na to ani nekouknuli.